lang/en/docs/package-json.md

```diff
@@ -363,6 +363,21 @@ If your package only allows one version of a given dependency, and you'd like to
 
 Note that if your `package.json` contains `"flat": true` and other packages depend on yours (e.g. you are building a library rather than an application), those other packages will also need `"flat": true` in their `package.json` or be installed with `yarn install --flat` on the command line.
```

Comment:
It may also be worth adding that running yarn install with `--flat` will auto-add resolutions for you